services/QuillLMS/client/app/bundles/Staff/components/comprehension/rulesAnalysis/ruleAnalysis.tsx

```diff
@@ -170,6 +170,7 @@ const RuleAnalysis = ({ history, match }) => {
       formattedResponse.response = tooltip
       formattedResponse.datetime = moment(r.datetime).format('MM/DD/YYYY')
       formattedResponse.strengthButtons = (<div className="strength-buttons">{strongButton}{weakButton}</div>)
+      formattedResponse.highlight = r.highlight
```

Comment:
This should be redundant, because `formattedResponse` will already have all the attributes of the response (see line 159).

Reply:
Thanks for pointing this out - it may be that the bug report is just due to a data issue.